asv/asv.conf.json

```diff
@@ -0,0 +1,75 @@
+{
+    // The version of the config file format.  Do not change, unless
+    // you know what you are doing.
+    "version": 1,
+
+    // The name of the project being benchmarked
+    "project": "pywt",
+
+    // The project's homepage
+    "project_url": "https://github.com/PyWavelets/pywt/",
+
+    // The URL or local path of the source code repository for the
+    // project being benchmarked
+    //"repo": "https://github.com/PyWavelets/pywt.git",
+    "repo": "..",
+
+    // List of branches to benchmark. If not provided, defaults to "master"
+    // (for git) or "tip" (for mercurial).
+    "branches": ["master"], // for git
+    // "branches": ["tip"],    // for mercurial
+
+    // The DVCS being used.  If not set, it will be automatically
+    // determined from "repo" by looking at the protocol in the URL
+    // (if remote), or by looking for special directories, such as
+    // ".git" (if local).
+    "dvcs": "git",
+
+    // The tool to use to create environments.  May be "conda",
+    // "virtualenv" or other value depending on the plugins in use.
+    // If missing or the empty string, the tool will be automatically
+    // determined by looking for tools on the PATH environment
+    // variable.
+    "environment_type": "virtualenv",
```

Comment:
If we comment out the line, we should mention that the user should uncomment and set to either `conda` or `virtualenv`.  When I had it commented out altogether `asv` issued a warning stating that this field would be required in a future release.